.meta/sources/socket.toml.erb

```diff
@@ -67,14 +67,15 @@ default = "host"
 description = """\
 The key name added to each event representing the current host. This can also \
 be globally set via the \
-[global `host_key` option][docs.reference.global-options#host_key].\
+[global 'host_key' option][docs.reference.global-options#host_key].\
```

Comment:
This seems to be a spurious change, is that intentional?

Reply:
Yes, this is intentional. `OptionLinker` change \`host_key`, so generated version looks incorrect.
https://github.com/timberio/vector/blob/1d8e88057f68d9cf9292ddc9edb69a7f8d3b3f92/scripts/generate/post_processors/option_linker.rb#L16
![Screenshot from 2020-05-24 08-12-51](https://user-images.githubusercontent.com/2633065/82746341-38fdc880-9d97-11ea-90e0-5f828b53c494.png)
Replace to single quote was quick solution, but probably I need fix regex in rb file :thinking: